### ISSUE TITLE
Search results: metadata display

### ISSUE THREAD

**Issue by eporter23** [emory-libraries/dlp-selfdeposit#438]
When a user performs a [search in the public UI](https://oe24-test.libraries.emory.edu/catalog?search_field=all_fields&q=) we want to configure the fields that display in the results page vs. what Hyrax provides by default.

While we may want to do further refinements on the layout in the future, as a first step we should adjust which fields display. 

**Fields to display (in the order noted below):** 

| Hyrax field          | SOLR field | Display Label | Notes |
| ----------- | ----------- | ----------- | ----------- |
| title      | title_tesim     |  Title | No change
| creator  | ?        | Authors | Please re-use what was implemented on the View Work page in #376
| date_issued  | date_issued_year_tesi        | Date |Full date entry; no hyperlink
| publisher  | publisher_tesim        | Publisher |Full date entry; no hyperlink
| publisher_version    | publisher_version_tesi        | Publication Version | Not sure if this will display as a link or not until we see it 
| license  | license_tesi        | License | Keep the hyperlink to the creative commons.org domain 

**Fields to remove:**

- [ ] All fields not specified above

**Comment by alexBLR** [comment]
@eporter23 is this for publications only?

**Comment by eporter23** [comment]
@alexBLR yeah. I'll review Collections separately and write up a separate ticket for that one. And FileSets shouldn't show up in the search results, so for this ticket it would just be for Publications.

**Comment by eporter23** [comment]
Looks great! Here's an example from oe24-test with all the data populated: <img width="1535" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBOG5mQmc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--2024e334f2a4139c0d051aa5ff770f42788352f9/Screenshot%202024-09-17%20at%201.52.35%E2%80%AFPM.png" alt="Screenshot 2024-09-17 at 1.52.35 PM.png" />

**Comment by eporter23** [comment]
@bwatson78 - apologies, but I specified the wrong SOLR date field for display. I'm adding a comment here mainly to remind myself to write a second ticket later.
